manifests/example-template.yml

```diff
@@ -1,40 +1,49 @@
-apiVersion: kubevirt.io/v1alpha1
-kind: VirtualMachineTemplate
+apiVersion: v1
+kind: Template
 metadata:
-  name: mytemplate
-spec:
-  template:
-    spec:
-      domain:
-        devices:
-          consoles:
-          - type: pty
-          disks:
-          - device: disk
-            driver:
-              cache: none
-              name: qemu
-              type: raw
-            snapshot: external
-            source:
-              host:
-                name: iscsi-demo-target.default
-                port: "3260"
-              name: iqn.2017-01.io.kubevirt:sn.42/2
-              protocol: iscsi
-            target:
-              dev: vda
-            type: network
-          graphics:
-          - type: spice
-          interfaces:
-          - source:
-              network: default
-            type: network
-          video:
-          - type: qxl
-        memory: 64MiB
-        os:
-          type:
-            os: hvm
-        type: qemu
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+  labels:
+    miq.github.io/kubvirt/os: rhel-7
+    miq.github.io/kubevirt-is-vm-template: true
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          cpu:
+            cores: ${CPU_CORES}
+          resources:
+            requests:
+              memory: ${MEMORY}
+          machine:
+            type: q35
+          devices:
+            disks:
+              - name: disk0
+                volumeName: disk0-pvc
+                disk:
+                  dev: vda
+        volumes:
+          - name: disk0-pvc
+            persistentVolumeClaim:
+              claimName: linux-vm-pvc-${NAME}
+- apiVersion: kubevirt.io/v1alpha1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: linux-vm-pvc-${NAME}
+parameters:
+- description: Name for the new VM
+  name: NAME
+- name: MEMORY
+  description: Amount of memory in MB
+  value: 4096Mi
```

Comment:
Same as above - descriptoij includes unit, but the unit is also part of value - but it can only be one of it.